packages/Switches/src/index.tsx

```diff
@@ -0,0 +1,136 @@
+import withUtils from "@blaze-react/utils";
+import React, { FunctionComponent, useState } from "react";
+
+interface IOptions {
+  checked: boolean;
+  value: string;
+  disabled: boolean;
+  required: boolean;
+  label: string;
+  id?: string;
+}
+interface ISwitchesProps {
+  labelPosition?: string;
+  options: IOptions[] | IOptions;
+  modifier?: string;
+  returnBoolean?: boolean;
+  onChange: ({
+    event,
+    checked,
+    data
+  }: {
+    event: React.ChangeEvent<HTMLInputElement>;
+    checked: IOptions[] | boolean;
+    data: IOptions[];
+  }) => void;
+  utils: {
+    uniqueId: (element: any) => string;
+    classNames: (className: string | object, classNames?: object) => string;
+  };
+}
+const Switches: FunctionComponent<ISwitchesProps> = ({
+  labelPosition,
```

Comment:
If you know the posible options of label position please do a constant as the PR before, we need to be this accessible from outside the component to pass it as a prop and avoid hardcoded strings.